figures/palettes/timepoint_color_palette.tsv

```diff
@@ -0,0 +1,5 @@
+color_names	hex_codes
+timepoint	dodgerblue3
```

Comment:
Just an fyi, these are not hexcodes. It doesn't affect the functionality, but would reccommend changing the column title or just adding the actual hexcode instead.

Reply:
This was a bit strange as I had it addressed here 713571b7e6263d4939ece528451714399a1d53aa
But I checked and edit again accordingly, see 1d6b677e43d9a6d542ff0abe6d21b25941e5d63e